package.json

```diff
@@ -281,6 +281,7 @@
     "@babel/preset-env": "7.8.7",
     "@babel/preset-react": "7.8.3",
     "@babel/preset-typescript": "7.16.5",
+    "@formatjs/icu-messageformat-parser": "2.0.19",
```

Comment:
Should this be a devDependency? How are these translations used in production?

Reply:
Yes, this is under the `devDependencies` in `package.json`. The both `ts-transformer` and `formatjs/cli` uses this dependency. So it was already in our `node_modules` just not used specifically.